api/dp-openapi.yaml

```diff
@@ -1158,6 +1158,8 @@ components:
         last_modified:
           type: string
           format: dateTime
+        username:
```

Comment:
Let's add a description that is something like "this field may not be included if the requestor is not authorized to see it".

Bonus: add a similar description for the `clearance` field in the `project` schema which I think is the only other place we change the schema based on privs.

Reply:
Done.